.changeset/brown-insects-double.md

```diff
@@ -0,0 +1,5 @@
+---
+"@turnkey/sdk-react": patch
+---
+
+Add passkeyConfig to EWK
```

Comment:
non blocking nit: can we add documentation in this PR (or the followup) that describes how to use this config?

essentially digging into

```
export interface PasskeyConfig {
  displayName?: string;
  name?: string;
}
```

a little self explanatory, but I think it's a low-effort way to make things a bit easier for devs! 😄